templates/rancher-airgap/overlay/opt/corral/rancher/install-rancher.sh

```diff
@@ -1,22 +1,42 @@
 #!/bin/bash
 set -ex
 
-helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
+if [ -z $CORRAL_rancher_chart_url ]; then
+  helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
+else
+  helm repo add rancher-latest $CORRAL_rancher_chart_url
+fi
+
 helm repo update
 
 CORRAL_internal_rancher_host=${CORRAL_internal_rancher_host:="${CORRAL_internal_fqdn}"}
 CORRAL_rancher_version=${CORRAL_rancher_version:=$(helm search repo rancher-latest/rancher -o json | jq -r .[0].version)}
 
-helm upgrade \
---install \
---create-namespace \
---set hostname="$CORRAL_internal_rancher_host" \
---set rancherImage="$CORRAL_registry_fqdn/rancher/rancher" \
---set systemDefaultRegistry="$CORRAL_registry_fqdn" \
---set useBundledSystemChart=true \
---version "${CORRAL_rancher_version}" \
---devel \
---wait \
-  -n cattle-system rancher rancher-latest/rancher
-  
+if [ -z $CORRAL_registry_cert ]; then
+  helm upgrade \
+  --install \
+  --create-namespace \
+  --set hostname="$CORRAL_internal_rancher_host" \
+  --set rancherImage="$CORRAL_registry_fqdn/rancher/rancher" \
+  --set systemDefaultRegistry="$CORRAL_registry_fqdn" \
+  --set useBundledSystemChart=true \
+  --version "${CORRAL_rancher_version}" \
+  --devel \
+  --wait \
+    -n cattle-system rancher rancher-latest/rancher
+else
+  helm upgrade \
+  --install \
+  --create-namespace \
```

Comment:
Is `--create-namespace` needed here if the namespace is created in `install-cert-manager.sh` ?

Reply:
This flag checks that the namespace is created first, so I'd like to keep it for posterity.

Reply:
agree with you Caleb